app/views/user/show/header.scala

```diff
@@ -209,7 +209,7 @@ It's useful against spambots. These marks are not visible to the public."""
                 )
               ),
               ctx.noKid option frag(
```

Comment:
looks like the checks could be factored out to this level, together with `ctx.noKid`